src/layouts/default.vue

```diff
@@ -20,7 +20,7 @@
     <VGlobalAudioSection />
   </div>
 </template>
-<script>
+<script lang="ts">
 import { computed, provide, ref, watch } from '@nuxtjs/composition-api'
```

Comment:
This file is missing the `defineComponent`.
```suggestion
<script lang="ts">
import { computed, provide, ref, watch, defineComponent } from '@nuxtjs/composition-api'
```

Reply:
Great catch!